client/components/seo/preview-upgrade-nudge/index.jsx

```diff
@@ -39,7 +39,8 @@ export const SeoPreviewNudge = ( {
 			<QueryPlans />
 			<TrackComponentView eventName="calypso_seo_preview_upgrade_nudge_impression" />
 
-			<Banner
+			<UpsellNudge
```

Comment:
I hadn't seen this one, thanks for tackling it! :D I've added it to our tracking list: #38778